### ISSUE TITLE
Command should print out usage information

### ISSUE THREAD

**Issue by wkoszek** [NARKOZ/gitlab#345]
I've installed a `gitlab` Gem, but the `gitlab help` isn't very helpful. I need to read a README.md from this repo to understand how to use it. I think `gitlab help` should print out a standard UNIX usage information with some examples.

**Comment by sotayamashita** [comment]
@wkoszek any example?

**Comment by NARKOZ** [comment]
> @wkoszek any example?

I think it should be like in `man hub` https://github.com/github/hub/blob/70a5493539d7a614449f0d32051e08b0dbc53bc3/share/man/man1/hub.1.ronn

**Comment by wkoszek** [comment]
@NARKOZ Right. Instead of just listing options, show 20 simple one-liners with explanations on what they do. `hub`, `awk`, `jq` and other commands will give you an idea. Thanks!
